src/aws.sqs/amazonsqs_errors.bal

```diff
@@ -25,54 +25,31 @@ public type ErrorDetail record {
 
 // Ballerina AWS SQS Client Error Types
 
-// Ballerina Basic Error Types
-
-public const HTTP_CLIENT_POST_REQUEST_FAILED = "{ballerinax/aws.sqs}HTTPClientPOSTRequestFailed";
-public type HTTPClientPOSTRequestFailed error<HTTP_CLIENT_POST_REQUEST_FAILED, ErrorDetail>;
-
 // AWS SQS Authenticator Error Types
 
-public const GENERATE_POST_REQUEST_FAILED = "{ballerinax/aws.sqs}GeneratePOSTRequestFailed";
-public type GeneratePOSTRequestFailed error<GENERATE_POST_REQUEST_FAILED, ErrorDetail>;
+public type GeneratePOSTRequestFailed distinct error;
 
 // AWS SQS Connector Error Types
 
-public const ERROR_CLIENT = "{ballerinax/aws.sqs}ErrorClient";
-public type ErrorClient error<ERROR_CLIENT, ErrorDetail>;
-
-public const ERROR_SERVER = "{ballerinax/aws.sqs}ErrorServer";
-public type ErrorServer error<ERROR_SERVER, ErrorDetail>;
-
-public const ERROR_OPERATION = "{ballerinax/aws.sqs}ErrorOperation";
-public type ErrorOperation error<ERROR_OPERATION, ErrorDetail>;
+public type ErrorOperation distinct error;
 
 // AWS SQS Data Mappings Error Types
 
-public const ERROR_DATA_MAPPING = "{ballerinax/aws.sqs}ErrorDataMapping";
-public type ErrorDataMapping error<ERROR_DATA_MAPPING, ErrorDetail>;
+public type ErrorDataMapping distinct error;
 
 public const CONVERT_XML_TO_OUTBOUND_MESSAGE_FAILED = "{ballerinax/aws.sqs}ConvertXmlToOutboundMessageFailed";
```

Comment:
Do we need these error reasons anymore?

Reply:
Removed all the unnecessary error reasons.